src/views/MainMenu.jsx

```diff
@@ -326,7 +326,11 @@ export default function MainMenu(props) {
 					</div>
 				</div>
 				{ocard() ?
-					<Card x={92} y={340} card={Cards.Codes[ocard()]} />
+					<>
+					<div style="font:16px sans-serif;user-select:none;position:absolute;pointer-events:none;overflow:hidden;z-index:5;left:95px;top:315px">Daily Login Reward</div>
+					<Card x={92} y={340} card={Cards.Codes[showcard]} />
+					{() => store.chatMsg('Daily Reward: ' + Cards.Codes[showcard].name, 'System')}
```

Comment:
This should be generated when ocard gets set, maybe as an effect, not as rendering